python/lsst/analysis/tools/analysisPlots/analysisPlots.py

```diff
@@ -329,3 +333,21 @@ def setDefaults(self):
         self.produce.panels["panel_flux"] = HistPanel()
         self.produce.panels["panel_flux"].label = "Psf/Ap Ratio"
         self.produce.panels["panel_flux"].hists = dict(fluxRatioMetric="Ratio")
+
+
+class SourcesPlot(AnalysisPlot, BaseSources):
+    """Base class for counting associated and unassociated sources."""
```

Comment:
I think this needs an updated docstring about the sorts of plots this supports or produces.

Reply:
Docstring updated to be more descriptive